stub-connector/src/main/java/uk/gov/ida/notification/stubconnector/resources/SendAuthnRequestResource.java

```diff
@@ -46,18 +46,44 @@ public SamlFormView setupAuthnRequest(@Session HttpSession session) throws Resol
                 AttributeConstants.EIDAS_DATE_OF_BIRTH_ATTRIBUTE_NAME
         );
 
-        String authnRequestId = UUID.randomUUID().toString();
+         return authnRequestGenerator.generate(
+                 UUID.randomUUID().toString(),
```

Comment:
Would you mind putting an underscore at the start of the request id?  Had to do this to connect to the hub:

https://github.com/alphagov/verify-proxy-node/compare/EID-1223-connect-proxy-node-to-hub#diff-0c000cbcc20f5ae7883478b90896131f